src/routes/components/components.json

```diff
@@ -2827,5 +2827,15 @@
 		"category": "Design System",
 		"tags": ["components and libraries", "component sets", "typescript"],
 		"stars": 0
+	},
+	{
+		"title": "simple-ui-components-in-svelte",
+		"url": "https://perfect-things.github.io/simple-ui-components-in-svelte/",
+		"description": "A set of simple & flexible UI components",
+		"npm": "simple-ui-components-in-svelte",
+		"addedOn": "2023-01-24",
+		"category": "Forms & User Input",
+		"tags": ["forms", "UI"],
```

Comment:
The tag UI doesn't exist

Reply:
That's a pity. It's a very useful tag 😉 
Removed.

Reply:
New tag can be added, but they need to be thoroughly thinked.
If a new tag is added, all other components need to be review to check if they can have it too.
And if almost all of them have it, the tag don't have a lot of meaning for the end user that want to filter (i.e. if adding the filter remove 2 items in a list of 300, it's worth nothing). And in the contrary, if a tag match almost nothing, it's either a very good filter or a terrible one.